magento/pwa-studio
# Clear Checkout Data on Logout

## Description

See #1638

## Related Issue





Closes #1638

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Run through repro steps in issue
2. Verify that data is removed from local storage on logout

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
